flopy/mf6/data/dfn/gwf-buy.dfn

```diff
@@ -1,4 +1,5 @@
 # --------------------- gwf buy options ---------------------
+# package-type stress-package
```

Comment:
The Buoyancy Package is not a stress package, so perhaps this terminology is not correct here.

Reply:
removed "stress-package" from buoyancy package dfn